desc/plotting.py

```diff
@@ -1776,7 +1893,7 @@ def plot_boozer_modes(eq, log=True, B0=True, num_modes=10, rho=None, ax=None, **
 
 
 def plot_boozer_surface(
-    eq, grid_compute=None, grid_plot=None, fill=True, ncontours=100, ax=None, **kwargs
+    eq, grid_compute=None, grid_plot=None, fill=False, ncontours=100, ax=None, **kwargs
```

Comment:
I made this change because that seems to be the more popular format in recent publications. It is easier to see how "straight" the contours are when they are not filled in. 

Can you update the baseline image for me? Whenever I do it, it fails the tests because I think I have the wrong version of matplotlib installed.